docs/testing/e2e_tests.md

```diff
@@ -46,8 +46,8 @@ Configure LoRa Basics Station and Network Server to run on your concentrator:
         ```bash
         NET_SRV_LOG_LEVEL=1
         NET_SRV_LOGTO_HUB=true
-        NET_SRV_LOGTO_UDP=true
-        NET_SRV_LOG_TO_UDP_ADDRESS=<your-local-ip-address>
+        NET_SRV_LOGTO_TCP=true
```

Comment:
This was changed to be consistent with `NET_SRV_LOG_TO_TCP_ADDRESS` and `NET_SRV_LOG_TO_TCP_PORT` where it's `LOG_TO` instead of `LOGTO`:

```suggestion
        NET_SRV_LOG_TO_TCP=true
```

Reply:
thanks for the catch!